packages/shared/src/constants.ts

```diff
@@ -39,3 +39,13 @@ export enum WALLETCONNECT_ERRORS {
 export const PRICES_REFRESH_INTERVAL = 60000
 
 export const TRANSACTIONS_REFRESH_INTERVAL = 10000
+
+// TIME
+
+export const ONE_DAY_MS = 1000 * 60 * 60 * 24
+
+export const ONE_HOUR_MS = 1000 * 60 * 60
+
+export const FIVE_MINUTES_MS = 1000 * 60 * 5
+
+export const ONE_MINUTE_MS = 1000 * 60
```

Comment:
nit: You'll hate me for this, but whatever, I'll propose it anyway and you do what you want with it 😆 

```ts
export const ONE_MINUTE_MS = 1000 * 60

export const FIVE_MINUTES_MS = 5 * ONE_MINUTE_MS

export const ONE_HOUR_MS = 60 * ONE_MINUTE_MS

export const ONE_DAY_MS = 24 * ONE_HOUR_MS
```

Reply:
hahahaha